core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -468,14 +468,6 @@ class KafkaController(val config: KafkaConfig,
 
     // shutdown leader rebalance scheduler
     kafkaScheduler.shutdown()
-    offlinePartitionCount = 0
```

Comment:
The way `updateMetrics` is written is a little annoying. While we're at it, would it be any clearer to revise `updateMetrics` to the following structure:
```scala
if (isActive) {
  offlinePartitionCount = controllerContext.offlinePartitionCount
  ...
} else {
  offlinePartitionCount = 0
  ...
}
```

Reply:
@hachikuji Thanks for your suggestion, I updated the code.